### PR TITLE
Added functionality to remove movie from favorites

### PR DESCRIPTION
**Pull request by BerkeleyReynolds**
I finished up adding the functionality to remove a movie from a user's favorites list. The button switches from "Add To Favorites" to "Favorited" depending on whether the movie is favorited or not.

### CODE REVIEW

**Review by benlyon03 — CHANGES_REQUESTED** [review]
Everything looks good Berkeley! But before you merge make sure to get rid of the unnecessary console.log() functions so we don't have to go back later and delete these since it is on our deployment branch.

**Review by shaikh7860 — APPROVED** [review]
Everything looks good!

**Comment by BerkeleyReynolds** [comment]
Ok, I will delete the unnecessary console.log calls and push the code right now. Thanks Ben

**Review by benlyon03 — APPROVED** [review]

File: frontend/src/Pages/Movie.js [hunk 1]
```diff
@@ -76,16 +77,16 @@ function Movie(props) {
     return newDate;
   }
 
-  const [favButtonText, setFavButtonText] = useState("Add to Favorites");
-  const [favButtonDisabled, changeDisabled] = useState(false);
+  const [favButtonText, setFavButtonText] = useState("Add To Favorites");
+  const [favButtonVariant, setFavButtonVariant] = useState("danger");
```
Comment (kibby111701) [inline frontend/src/Pages/Movie.js:81]: Is the variant ever not using danger? I don't see where this is used

**Review by kibby111701 — APPROVED** [review]

File: frontend/src/Pages/Movie.js (hunk 1 above)
Comment (BerkeleyReynolds) [reply frontend/src/Pages/Movie.js:81]: I was trying to make the button change colors when it switches from "Add To Favorites" to "Favorited". That is the purpose of coding it in this fashion.

**Pull request merged**